docs/source/getting_started/code_examples/plot_image_augmentations.py

```diff
@@ -0,0 +1,24 @@
+import glob
+from PIL import Image
+import lightly
+
+# let's get all jpg filenames from a folder
+glob_to_data = '/datasets/clothing-dataset/images/*.jpg'
+fnames = glob.glob(glob_to_data)
+
+# load the first two images using pillow
+input_images = [Image.open(fname) for fname in fnames[:2]]
```

Comment:
I think you could also use lightly dataset to load the images instead:

```
dataset = LightlyDataset('/datasets/clothing-dataset/images')
input_images = [dataset[index][0] for index in range(2)]
```

Reply:
I know, but I thought about going as simple as possible for that one so people could also use it outside of a `LightlyDataset`